DH_Engine/Classes/DHProjectileWeapon.uc

```diff
@@ -153,10 +153,15 @@ var     bool            bCanUseIronsights;      // allows firing from a shoulder
 
 enum EWeaponComponentAnimationDriverType
 {
+    DRIVER_None,                // Not driven by anything other than animation notifies.
```

Comment:
This can actually be removed, it was a WIP thing that I left in by mistake

Reply:
Resolved in latest commit